### ISSUE TITLE
ava Swing Ayatana works perfectly with ide installed with Umake 

### ISSUE THREAD

**Issue by dggonzalez1971** [ubuntu/ubuntu-make#62]
Java Swing Ayatana (JAyatana) is a project that integrates Java Swing applications with Ubuntu's global menu (AppMenu) and HUD. Works perfectly with Eclipse, PyCharm and Idea 
In Ubuntu 14.10
sudo add-apt-repository ppa:danjaredg/jayatana
sudo apt-get update
sudo apt-get install jayatana
In Ubuntu 15.04
sudo apt-get install jayatana
You must reboot the computer
https://code.google.com/p/java-swing-ayatana/

**Comment by Tinche** [comment]
Hi,

http://www.webupd8.org/2015/01/ubuntu-1504-might-get-global-menu-and.html

There's been talk of including this, but looks like it might be unnecessary soon.

**Comment by didrocks** [comment]
@dggonzalez1971: Thanks for the suggestion!

For your information, we work closely with upstream since December for this. jayatana has been in the ubuntu repo as I pushed it and will be in main (so installed by default on ubuntu-desktop) in the next couple of days.

You might as well notice that for older releases, I pushed yesterday https://github.com/ubuntu/ubuntu-make/commit/b19dd0aa68773888d3a2641b42a680d98a0b6825 and the package are in the ubuntu-make ppa: https://launchpad.net/~ubuntu-desktop/+archive/ubuntu/ubuntu-make!

So yeah, just stay tuned! :)
